File: URI Online Judg/Beginner/URI-Python/1038.py
num = input().split(' ')

a, b = num

a, b = int(a), float(b)

if a == 1:
    print("Total: R$ %.2f"%(4.00 * b))

elif a == 2:
    print("Total: R$ %.2f"%(4.50 * b))

elif a == 3:
    print("Total: R$ %.2f"%(5.00 * b))

elif a == 4:
    print("Total: R$ %.2f"%(2.00 * b))

elif a == 5:
    print("Total: R$ %.2f"%(1.50 * b))

File: URI Online Judg/Beginner/URI-C-Plus-Plus/1038.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(int argc, char const *argv[]) {
	int quantity, num;
	std::cin >> num >> quantity;

	if (num == 1){
		std::cout << "Total: R$ " << fixed << setprecision(2) << 4.00 * quantity << '\n';
	}

	else if (num == 2){
		std::cout << "Total: R$ " << fixed << setprecision(2) << 4.50 * quantity << '\n';
	}

	else if (num == 3){
		std::cout << "Total: R$ " << fixed << setprecision(2) << 5.00 * quantity << '\n';
	}

	else if(num == 4){
		std::cout << "Total: R$ " << fixed << setprecision(2) << 2.00 * quantity << '\n';
	}
	else if (num == 5) {
		std::cout << "Total: R$ " << fixed << setprecision(2) << 1.50 * quantity << '\n';
	}

	return 0;
}
